etgtools/extractors.py

```diff
@@ -533,10 +533,14 @@ def makePyArgsString(self):
                         default = '|'.join([self.cleanName(x, True) for x in default.split('|')])
                         s = f'{s}={default}'
                     elif param_type:
-                        s = f'{s} : {param_type}'
+                        s = f'{s}: {param_type}'
                     params.append(s)
 
         self.pyArgsString = f"({', '.join(params)})"
+        # __bool__ and __nonzero__ need to be defined as returning int for SIP, but for Python
```

Comment:
`__nonzero__` is python 2, is there still references to that?

Reply:
Yeah, the sip code still adds a `__nonzero__` to anything with a `__bool__`.  I didn't remove those since that isn't the focus of this PR.